db/migrate/20220606192842_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :keywords
+      t.string :name
```

Comment:
Can we include some validations so we can test them in the future? I'm thinking on:
* Not null for both cases
* Not empty for both cases
* Keyword is a CSV
* And a complex/weird one so we also have a custom validation, e.g. name should start with a capital letter or another one you would like, as long it has some custom logic I'm fine


Also, can we use some callback here? e.g. to remove trailing and starting white spaces? Or maybe populating a computed columns called `keywrods_count` 😬